app/assets/stylesheets/custom.css.scss

```diff
@@ -108,3 +108,46 @@ footer {
 
 }
 
+/* sidebar */
+
+aside {
+
+  section.user_info {
+    margin-top: 20px;
+  }
+
+  section {
+    padding: 10px 0;
+    margin-top: 20px;
+
+    &:first-child {
+      border: 0;
+      padding-top: 0;
+    }
+
+    span {
+      display: block;
+      margin-bottom: 3px;
+      line-height: 1;
+    }
+
+    h1 {
+      font-size: 1.4em;
+      text-align: left;
```

Comment:
Properties should be ordered font-size, letter-spacing, margin-bottom, margin-top, text-align